zkevm-circuits/src/witness/block.rs

```diff
@@ -74,6 +74,14 @@ pub struct Block<F> {
     pub start_l1_queue_index: u64,
     /// IO to/from precompile calls.
     pub precompile_events: PrecompileEvents,
+    /// Previous last applied l1 block
+    pub prev_last_applied_l1_block: Option<u64>,
+    /// Last applied l1 block
+    pub last_applied_l1_block: Option<u64>,
+    /// L1 block hashes
+    pub cum_l1_block_hashes: Vec<Hash>,
+    /// L1 block range hash
+    pub l1_block_range_hash: Option<Hash>,
```

Comment:
similar to the comment in `bus-mapping` crate's changes. You don't need this field and can compute it from the above field, i.e. `cum_l1_block_hashes`.